src/app/pages/mass-renewal/mass-renewal.hbs

```diff
@@ -0,0 +1,163 @@
+                <input name="00NU00000052VId" type="number" required />
+            </label>
+
+            <label>Which partner resources are you using with your book?
+                <select multiple name="00NU0000005VmTu">
+                     <option value="None" disabled selected>None</option>
+                     <option value="Cerego">Cerego</option>
+                     <option value="Courseweaver">Courseweaver</option>
+                     <option value="Expert TA">Expert TA</option>
+                     <option value="Lrnr">Lrnr</option>
+                     <option value="Sapling Learning">Sapling Learning</option>
+                     <option value="SimBio">SimBio</option>
+                     <option value="WebAssign">WebAssign</option>
+                     <option value="Wiley Plus">Wiley Plus</option>
+                     <option value="XYZ Homework">XYZ Homework</option>
+                     <option value="" data-other="other-partner">Other (specify below)</option>
+                </select>
+            </label>
+
+            <label id="other-partner">Other
+                <input name="00NU0000005Vls8" class="short" type="text"
+                placeholder="Partner resources not listed above?">
+            </label>
+
+            <label class="question">
+                Which of our partners would you like to give permission to contact you about additional resources to support our books?
+                <select multiple name="00NU00000055spm">
+                    <option value="None" disabled selected>None</option>
+                    <option value="Online homework partners">Online homework partners</option>
+                    <option value="Adaptive courseware partners">Adaptive courseware partners</option>
+                    <option value="Customization tool partners">Customization tool partners</option>
+                </select>
+            </label>
+        </div>
+
+        <div class="col">
+            <!-- Doesn't look like Salesforce can store this information
+            <p>
+                Do you know of any other instructors using OpenStax books?
+            </p>
+            <fieldset class="labeled-inputs">
+                <label>
+                    <span>First name</span>
+                    <input name="adoption[hint_first_name]" type="text">
+                </label>
+                <label>
+                    <span>Last name</span>
+                    <input name="adoption[hint_last_name]" type="text">
+                </label>
+                <label>
+                    <span>Email address</span>
+                    <input name="adoption[hint_email_address]" pattern="[^@]+@([^@\.]+\.)+[^@\.]+" type="text">
+                </label>
+                <label>
+                    <span>Phone number</span>
+                    <input name="adoption[hint_phone_number]" type="text">
+                </label>
+            </fieldset>
+            -->
```

Comment:
Using Handlebars-style comments will keep this from ending up in the production DOM (`{{!-- --}}`).